assets/templates/aws.vpcflow/schema-a/gotext.tpl

```diff
@@ -1 +1,6 @@
-{{generate "Version"}} {{generate "AccountID"}} {{generate "InterfaceID"}} {{generate "SrcAddr"}} {{generate "DstAddr"}} {{generate "SrcPort"}} {{generate "DstPort"}} {{generate "Protocol"}}{{ $packets := generate "Packets" }} {{ $packets }} {{mul $packets 15 }} {{$startOffset := generate "StartOffset" }}{{$startOffsetInSecond := mul -1 1000000000 $startOffset }}{{$startOffsetDuration := timeDuration $startOffsetInSecond}}{{$end := generate "End" }}{{$start := $end.Add $startOffsetDuration}}{{$start.Format "2006-01-02T15:04:05.999999Z07:00" }} {{$end.Format "2006-01-02T15:04:05.999999Z07:00"}} {{generate "Action"}}{{ if eq $packets 0 }} NODATA {{ else }} {{generate "LogStatus"}} {{ end }}
\ No newline at end of file
+{{- $startOffset := generate "StartOffset" }}
+{{- $startOffsetInSecond := mul -1 1000000000 $startOffset }}
+{{- $startOffsetDuration := $startOffsetInSecond | int64 | duration}}
```

Comment:
`$startOffsetInSecond` was indeed `$startOffsetInNanosecond` (my bad :))
`$startOffset` was multiplied `1000000000` because our `timeDuration` expected nanoseconds, `duration` from sprig expects seconds: so we need to use directly the value in `$startOffset` (still multiplied by `-1` since we want to subtract the value) is already the correct one.
Not sure if you prefer a oneliner or not, but this might work as well:
```
{{- $start := mul -1 $startOffset | int64 | duration | $end.Add }}
```

Reply:
Thanks for noticing, I completely missed that.
I updated the template.

Reply:
my oneliner was wrong:
`time.Time.Add` takes a `time.Duration`, but `duration` returns a string, so we must use `dateModify` from sprig
```
{{- $start := $end | dateModify  (mul -1 $startOffset | int64 | duration) }}
```

todo: let's add to CI to generate at least one event for all the templates